plugins/status/plugin.go

```diff
@@ -204,7 +205,9 @@ func New(parsedConfig *Config, manager *plugins.Manager) *Plugin {
 		decisionLogsCh: make(chan lstat.Status),
 		stop:           make(chan chan struct{}),
 		reconfig:       make(chan reconfigure),
-		pluginStatusCh: make(chan map[string]*plugins.Status),
+		// we use a buffered channel here to avoid blocking other plugins
+		// when updating statuses
+		pluginStatusCh: make(chan map[string]*plugins.Status, 3),
```

Comment:
This is the other part of the fix, https://github.com/open-policy-agent/opa/issues/6343#issuecomment-1780266088

Reply:
Curious how we can up with the `3`?

Reply:
Actually, all we need to address the issue is 1, so I have used that instead in 908b57067100420fa2ae6f835bf360ccb41fe22d